bucket/trufflehog.json

```diff
@@ -0,0 +1,24 @@
+{
+    "version": "3.4.3",
+    "description": "Find leaked credentials",
+    "homepage": "https://github.com/trufflesecurity/trufflehog",
+    "license": "AGPL-3.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/trufflesecurity/trufflehog/releases/download/v3.4.3/trufflehog_3.4.3_windows_amd64.tar.gz",
+            "hash": "e10b28dfc388d0ba8bdb6e84187f6b90ce3b86238f61f31bec2c07394876d895"
+        }
+    },
+    "bin": "trufflehog.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/trufflesecurity/trufflehog/releases/download/v$version/trufflehog_$version_windows_amd64.tar.gz"
+            }
+        },
+        "hash": {
+            "url": "https://github.com/trufflesecurity/trufflehog/releases/download/v$version/trufflehog_$version_checksums.txt"
```

Comment:
```suggestion
            "url": "$baseurl/trufflehog_$version_checksums.txt"
```

Reply:
Thanks for the tip! I didn't know about it...